test/test_cpu/test_export.py

```diff
@@ -558,9 +558,7 @@ def test_nvfp4_moe_actmax_ar(self):
             nsamples=2,
             dataset=self.llm_dataloader,
         )
-        autoround.quantize()
-        quantized_model_path = self.save_dir
-        autoround.save_quantized(output_dir=quantized_model_path, inplace=False, format="auto_round")
+        autoround.quantize_and_save(output_dir=self.save_dir, inplace=True, format="auto_round")
```

Comment:
change one ut to quantize, as only quantize could expose the issue you are triyng to fix